src/journal-remote/journal-upload.c

```diff
@@ -9,8 +9,13 @@
 
 #include "sd-daemon.h"
 
+#include "math.h"
+#include "unit-printf.h"
```

Comment:
Please drop unit-printf.h and load-fragment.h, and sort headers.
Also, math.h is a system header. Please include it earlier with `<>`.